src/NServiceBus.Core.Tests/Recoverability/SecondLevelRetries/SecondLevelRetriesTests.cs

```diff
@@ -147,6 +177,10 @@ public void AbortReceiveOperation()
                 ReceiveOperationWasAborted = true;
             }
         }
+        static FailureInfoStorage GetFailureInfoStorage()
```

Comment:
My argument are:
 * From test perspective `10` in constructor is nose
 * We reduce coupling. If constructor changes we have a better change of changing only one line instead of all usages in tests